components.html

```diff
@@ -357,24 +362,54 @@ <h3>
+                    style="width: 563px; height: 858px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
suggestion : can use the separate class for iframe for styling. so that you can reuse it.

Reply:
good call, will try doing this.